### PR TITLE
[PUI] Make API form Select Field theme-aware

### PR DESCRIPTION
**Pull request by LavissaWoW**
Adding theme awareness to `react-select` field in forms.

Closes https://github.com/inventree/InvenTree/issues/5836

### CODE REVIEW

**Comment by matmair** [comment]
Seems to be the case 
grafik

**Comment by matmair** [comment]
Closes https://github.com/inventree/InvenTree/issues/5836

**Comment by LavissaWoW** [comment]
Yes :>

**Comment by matmair** [comment]
@LavissaWoW could you also take a look at https://github.com/inventree/InvenTree/issues/5824?

**Comment by LavissaWoW** [comment]
@matmair Can do!

**Comment by SchrodingersGat** [comment]
Cheers @LavissaWoW , LGTM

File: src/frontend/src/components/forms/fields/RelatedModelField.tsx [hunk 1]
```diff
@@ -209,8 +209,54 @@ export function RelatedModelField({
     [pk, data]
   );
 
+  // Field doesn't follow Mantine theming
+  // Define color theme to pass to field based on Mantine theme
+  const th = useMantineTheme();
+  let colors: any;
```
Comment (wolflu05) [inline src/frontend/src/components/forms/fields/RelatedModelField.tsx:215]: Same thing here, it would make sense to wrap this into a useMemo

**Pull request merged**